README.md

```diff
@@ -83,3 +83,87 @@ A script to do some basic QC is provided in `bin/qc.py`. It measures the % of re
 
 #### Output
 A subdirectory for each process in the workflow is created in `--outdir`. A `nml_upload` subdirectory containing dehosted fastq files and consensus sequences is included. 
+
+#### Provenance
+In the output directory for each sample, a provenance file will be written with the following format:
+```
+- process_name: performHostFilter
+  tools:
+    - tool_name: bwa mem
+      tool_version: 0.7.17-r1188
+      subcommand: mem
+      parameters:
+        - parameter: -t
+          value: 8
+    - tool_name: samtools
+      tool_version: 1.10
+      subcommand: sort
+- process_name: readTrimming
+  tools:
+    - tool_name: trim_galore
+      tool_version: 0.6.4_dev
+- process_name: filterResidualAdapters
+  tools:
+    - tool_name: filter_residual_adapters.py
+          sha256: c3d062687abf2bbec48721a562ec609742101eec82887b1f31b9997361da901e
+- process_name: trimPrimerSequences
+  tools:
+    - tool_name: samtools
+      tool_version: 1.10
+      subcommand: view
+      parameters:
+        - parameter: -F4
```

Comment:
I think this should be:

```
- parameter: -F
  value: 4
```

see: https://www.htslib.org/doc/samtools-view.html

Reply:
fixed!